libs/application/templates/criminal-record/src/forms/Payment.tsx

```diff
@@ -0,0 +1,75 @@
+import { m } from '../lib/messages'
+import {
+  buildForm,
+  buildDescriptionField,
+  buildSection,
+  Form,
+  FormModes,
+  buildCustomField,
+  buildMultiField,
+} from '@island.is/application/core'
+
+export const Payment: Form = buildForm({
+  id: 'DrivingLicenseApplicationPaymentForm',
+  title: '',
+  mode: FormModes.APPLYING,
+  renderLastScreenButton: false,
+  children: [
+    buildSection({
+      id: 'externalData',
+      title: m.externalDataSection,
+      children: [],
+    }),
+
+    buildSection({
+      id: 'awaitingPayment',
+      title: m.payment,
+      children: [
+        buildDescriptionField({
+          id: 'infoAwaitingPayment',
+          title: m.payment,
+          condition: () => {
+            return !window.document.location.href.match(/\?done$/)
+          },
+          description: (application) => {
+            const { paymentUrl } = application.externalData.createCharge
+              .data as { paymentUrl: string }
+
+            if (!paymentUrl) {
+              throw new Error()
```

Comment:
I think this will break the application and likely require manual database state change so that the user can interact with it again.

Reply:
You might want to conditionally show a screen in case the url is missing to allow the user to move the application to draft and try again. Similarly only showing this screen when the url does exist.

Reply:
Okay thanks, fixing it :)